lib/ListTemplate/listTemplate.js

```diff
@@ -9,6 +10,10 @@ export const listTemplate = {
     return <FormattedDate value={updatedDate} />;
   },
   updatedBy: record => {
+    if (!record.userInfo) {
+      return <NoValue />;
```

Comment:
Per comment from PO the dash is not needed when the is no data to display.

```suggestion
      return null;
```